examples/Client/PublishSubscribe/BulkPublishEventExample/README.md

```diff
@@ -0,0 +1,43 @@
+# Dapr .NET SDK Bulk publish example
+
+## Prerequisites
+
+- [.NET Core 3.1 or .NET 5+](https://dotnet.microsoft.com/download) installed
```

Comment:
should this be modified to latest .NET?

Reply:
I guess this means we support all .NET versions after 5 unless I'm misunderstanding something.

Reply:
For the main libs, you need 3.1. For testing we support 3.1, 5, and 6. Though I have a PR out that removes 5 and adds 7.